docs-raw/source/installation.rst

```diff
@@ -0,0 +1,58 @@
+.. _installation:
+
+Installation
+==============
+
+**Linux**
+
+On most popular distributions Pure Maps can
+simply be installed via package manager. 
+
+.. image:: https://repology.org/badge/vertical-allrepos/pure-maps.svg
+   :alt: Packaging status
+   :target: https://repology.org/project/pure-maps/versions
+
+
+For example on Alpine/postmarketOS:
+
+.. code-block:: shell
+
+ sudo apk add pure-maps osmscout-server
+ 
+
+**Flatpak**
+
+There is a Flatpak build on `flathub`_.
+
+.. code-block:: shell
+
+ sudo flatpak install flathub io.github.rinigus.PureMaps
+ sudo flatpak install flathub io.github.rinigus.OSMScoutServer
+ 
+.. _flathub: https://flathub.org/apps/io.github.rinigus.PureMaps
+
+**UBports**
+
+Install via `open-store`_
+
+.. _open-store: https://open-store.io/app/pure-maps.jonnius
```

Comment:
You can also mention the alternative https://open-store.io/app/pure-maps-slim.jonnius, which only differs in not shipping the English-only high quality TTS to reduce the package size by the factor of 10.